citrixitm/config.go

```diff
@@ -9,6 +9,13 @@ import (
 	"golang.org/x/oauth2/clientcredentials"
 )
 
+const (
+	libraryName     = "terraform-provider-citrixitm"
+	libraryVersion  = "0.1.0"
```

Comment:
does the version need to be bumped up to a 1.x?

Reply:
Yes! Thanks for spotting that.

Reply:
I'll likely bump that after I merge all the outstanding PRs and make a release tag, which is when the new version number is born.